### ISSUE TITLE
[wg/svg]  SVG Working Group

### ISSUE THREAD

**Issue by plehegar** [w3c/strategy#432]
~~Lack of charter proposal, reviewers please take note.~~

# Charter Review

~~No charter. this is a WG closure.~~

We're drafting a new charter.

[charter](https://w3c.github.io/charter-drafts/2024/svg-wg.html)

[diff from previous charter](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2022%2F02%2Fsvg-wg-charter.html&doc2=https%3A%2F%2Fw3c.github.io%2Fcharter-drafts%2F2024%2Fsvg-wg.html)

What kind of charter is this? Check the relevant box / remove irrelevant branches. 

"There are insufficient member resources to produce chartered deliverables or to maintain the group, according to priorities established within W3C." [4.6. Chartered Group Closure](https://www.w3.org/2023/Process-20230612/#GeneralTermination)

There is no charter. The Group does not have the resources necessary to maintain the SVG specification properly.

SVG is used widely on the Web nowadays but there is a lack of interest of maintaining the specification, including re-aligning the specification with the HTML specification.

Anything else we should think about as we review? 

Should we do an AC review for this or not? [Process](https://www.w3.org/2023/Process-20230612/#GeneralTermination) doesn't ask us to do so.

cc @caribouW3 @dirkschulze

**Comment by plehegar** [comment]
If the AC review is negative and we still don't have resources, what would we do?

**Comment by chris-little** [comment]
As a very passive member of SVG WG and a user rather than developer, I have to say the SVG Recommendations meet most of my interoperability needs for 2D graphics (e.g. weather maps (x,y), cross-sections (x,z), Hovmuller diagrams (z,t), lots of simple but obscure symbols, blended imagery and line graphics, etc),
The one area where I would like improvement is not purely SVG: better integration and handling of Unicode glyphs, fonts and symbols, whether using a Unicode PUA (Private Use Area) or not.
That is probably not helpful, but not sure where the issue could be tackled.

**Comment by r12a** [comment]
>  The one area where I would like improvement is not purely SVG: better integration and handling of Unicode glyphs, fonts and symbols, whether using a Unicode PUA (Private Use Area) or not.

@chris-little can you give an example of the problem(s) you are facing?  (It's not clear to me why this is an issue.)

**Comment by chris-little** [comment]
@r12a I suspect it is probably more of a tooling issue, but a typical weather application may want to place [many specialised symbols](https://github.com/OGCMetOceanDWG/WorldWeatherSymbols/tree/master/symbols), combined with conventional text glyphs in [complicated, non-linear layouts](https://www.dwd.de/EN/ourservices/hobbymet_wcharts_europe/hobbyeuropecharts.html). International aviation also has widespread use of similar requirements in their [SigWx charts](https://www.aviationweather.gov/data/iffdp/3103.pdf)

Some other examples are:
 https://www.netweather.tv/charts-and-data/fax or 
https://meteologix.com/uk/observations/weather-observation.html or 
https://www.metlink.org/wp-content/uploads/2020/12/Fig_5.gif 

The issue is the inconsistent support for the mixture of symbols, text treated as symbols, fonts (and speed of rendering, causing  developers to create symbols or explicit graphics of conventional letter glyphs, undermining search) and inconsistent default sizing and positioning. I suspect HTML and CSS rendering may have an impact too.

My wish is too reduce the cost of development and display of the above types of charts/maps. I apologise if it seems a rather generic whinge, but I recognise that I may need to discuss with some current developers to see if they still echo my concerns.

**Comment by plehegar** [comment]
Discussed at a recent AB-led session: https://www.w3.org/2023/10/17-ac-minutes.html#t13

**Comment by chris-little** [comment]
@plehegar Apologies, but I cannot access that page, probably because we have just chosen to let our W3C membership lapse.

**Comment by plehegar** [comment]
Out that conversation came one idea: create a Maintenance Working Group in charge of hosting Core Web specifications that don't have a dedicated group to maintain. We certainly have a few of them but SVG is the posterchild example of such specification. To avoid Patent Policy issues, the specifications would be added one by one in it as the need arises. cc @tantek

**Comment by plehegar** [comment]
An other alternative would be to have a Maintenance Community Group instead of a Working Group. This would limit the IP exposure of the participants, make it easier for someone from the Community to step up, but prevent the introduction of new features in the specifications.

**Comment by astearns** [comment]
Capturing some ideas from a side conversation this week (these are not all my ideas)

SVG maintenance group with reduced scope:
* improve interoperability and specification/test quality and completeness for the existing features
* improved integration with CSS and HTML for animations, styling, layout and semantics
* no new features
* reduced overhead (mainly use async communication, hold meetings only when there is a sufficient agenda)
* participation commitments from at least 2 browser implementer teams

It's important to republish SVG2 CR as a CRS against the new Patent Policy before the current group closes, because its current latent patent commitments won't be realized unless the group does so

**Comment by chrishtr** [comment]
> * participation commitments from at least 2 browser implementer teams

I'd prefer all 3. @zcorpan WDYT?

**Comment by plehegar** [comment]
Conclusion: have an SVG Working Group with the proposed scope above.

**Comment by plehegar** [comment]
(note: for those who will want new features, we could always recharter in the future)

**Comment by caribouW3** [comment]
https://w3c.github.io/charter-drafts/2024/svg-wg.html
(please comment in https://github.com/w3c/charter-drafts/issues )

**Comment by plehegar** [comment]
Advance notice: https://lists.w3.org/Archives/Public/public-new-work/2024Feb/0002.html

**Comment by plehegar** [comment]
cc @bkardell

**Comment by himorin** [comment]
- last two entries in history table (last rechartering and extension) has no valid link

**Comment by ruoxiran** [comment]
APA is fine with this charter.

**Comment by himorin** [comment]
no comment or request from i18n

(sorry for late, due that I've been on business trip..)

**Comment by plehegar** [comment]
No comments from PING

**Comment by simoneonofri** [comment]
No comments from the Security side.

Maybe in the future, it should be interesting to note the scenario of [Stored XSS using SVG](https://hackerone.com/reports/148853).

**Comment by siusin** [comment]
Perhaps we should remove `the Director` from _Sec 8. Decision Policy_?

**Comment by himorin** [comment]
section number of decision policy in section 8 is old.

**Comment by caribouW3** [comment]
I updated section 8 to remove mention of Director and correct the section number.

**Comment by plehegar** [comment]
AC Review: https://lists.w3.org/Archives/Member/w3c-ac-members/2024AprJun/0011.html
Deadline is May 23/24

**Comment by iadawn** [comment]
There is a desire from some Members to see SVG Accessibility API Mappings completed and published. This has the potential to significantly improve the accessibility of SVG and it would be good if there was an expected completion date. If there is a lack of resource to move it forward could we either work with one of the a11y WGs to take on the work or move it into there to complete it?

**Comment by chrishtr** [comment]
> There is a desire from some Members to see SVG Accessibility API Mappings completed and published.

Is there work you can refer to that enumerates which parts are missing?

**Comment by iadawn** [comment]
> Is there work you can refer to that enumerates which parts are missing?

This isn't something that I can answer. Based on the feedback I received, it was more a desire to see this published as a Recommendation.

**Comment by plehegar** [comment]
Deadline since we extended the charer is June 7.

**Comment by dontcallmedom** [comment]
Group announced https://lists.w3.org/Archives/Public/public-new-work/2024Jun/0012.html
